### ISSUE TITLE
is-about

### ISSUE THREAD

**Issue by wceusters** [HumanBehaviourChangeProject/ontologies#7]
In the paper https://wellcomeopenresearch.org/articles/6-77/v1 I read " The relationship ‘is_about’ from the Information Artifact Ontology (Ceusters, 2012) was also used to represent one class presenting information about another (e.g Total number of people able to deliver intervention is_about Source)." I don't find this relational expressed in the BCIO version here, so perhaps it is not an issue. But note that if you intend to do so as in the paper, it wouldn't work under the BFO principles: since 'source' is defined in the paper as 'role played by a person', so you would be specifying how many distinct intervention roles one specific person has. Linking it to 'person' would also not work, as the total number of an instance of 'person' is 1. The range of the relation in your example would be an aggregate of persons.

**Comment by jannahastings** [comment]
@wceusters, many apologies for the delay responding, for some reason I was not notified about this comment, thank you! 

You are absolutely correct, the target of the 'is about' relation for an entity 'total number of people able to deliver intervention' should be an aggregate of persons, not an individual person nor a role. We will correct that in the next release.

**Comment by robertjwest** [comment]
Good catch - thanks. To to it correctly then I think we need a defined class for a person who has the role of source, rather like in a previous comment we need a defined class for 'occupation holder' (though we may want to keep our 'occupational role' as the primary class for this rather than thinking of occupation as a process aggregate?

**Comment by mayabraun** [comment]
Thanks again for pointing this out to us. We have tackled the issue, and it will be resolved in the next release.
